roles/sap_vm_provision/tasks/platform_ansible/ovirt_vm/execute_main.yml

```diff
@@ -30,6 +30,16 @@
       vars:
         ovirt_auth: "{{ sap_vm_provision_ovirt_auth }}"
 
+    - name: Set fact when using Bastion SSH Proxy connection from Ansible control node to target node/s
+      ansible.builtin.set_fact:
+        __ssh_args_bastion: -o ProxyCommand='ssh -W %h:%p {{ sap_vm_provision_bastion_user }}@{{ sap_vm_provision_bastion_public_ip }} -p {{ sap_vm_provision_bastion_ssh_port }} -i {{ sap_vm_provision_ssh_bastion_private_key_file_path }} -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null'
```

Comment:
@sean-freeman I think it would make sense to make the ssh_args a global variable, wdyt? Otherwise it would be hard to change, e.g. `StrictHostKeyChecking=no` without a code change.